static/src/stylesheets/layout/new-header/_pillar-link.scss

```diff
@@ -5,7 +5,7 @@
     cursor: pointer;
     display: block;
     font-size: 16px;
-    font-size: 5.1vw;
+    font-size: 4.9vw;
```

Comment:
Yeah, that could be one possible solution. Will add the to the list of things to consider aswell as the animation hint you gave. 👍